### DISCUSSION TITLE
carbon stops writing metrics

### DISCUSSION THREAD

**Discussion by bcwazoku** [grafana/grafana#44634] [General]
Hi,
 After a few days or working properly, carbon randomly stops writing metrics to disk. The OS and the disks are otherwise fine. Restarting carbon-cache brings things back to normal.
Am I doing anything wrong in the configuration or is this a known issue ? 
Could somene help me please on how to fix this.
Attached is the log of console while it errors [carbon.txt](https://github.com/grafana/grafana/files/7968909/carbon.txt)
I am using 
graphite-carbon                       1.1.4-2

on debian 10 (buster)

**Comment by ifrost** [comment]
Hi @bcwazoku. This looks like a carbon specific issue. I think https://github.com/graphite-project/carbon might be a better place to ask this question. Grafana GitHub Discussions are used to discuss Grafana feature requests.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
